packages/aws_bedrock/data_stream/runtime/manifest.yml

```
@@ -0,0 +1,61 @@
+title: AWS Bedrock Runtime Metrics
+type: metrics
+elasticsearch:
+  index_mode: time_series
+streams:
+  - input: aws/metrics
+    title: AWS Bedrock metrics
+    description: Collect AWS Bedrock metrics
+    vars:
+      - name: period
+        type: text
+        title: Collection Period
+        multi: false
+        required: true
+        show_user: true
+        default: 5m
+      - name: data_granularity
+        type: text
+        title: Data Granularity
+        multi: false
+        required: false
+        show_user: false
+      - name: regions
+        type: text
+        title: Regions
+        multi: true
+        required: false
+        show_user: true
+      - name: latency
+        type: text
+        title: Latency
+        multi: false
+        required: false
+        show_user: false
+        description: To address latency issues between certain AWS services and CloudWatch, specify a latency parameter to adjust the collection start time and end time in Metricbeat such as `15m`.
```

The description written by @lucian-ioan might provide a bit more context about this parameter.

```suggestion
        description: The 'latency' parameter adjusts the Metricbeat collection start and end times. AWS CloudWatch might experience delay in processing metrics for some services causing data points to be missed during the integration collection period. To mitigate this potential issue, specify a latency parameter such as `15m`.
```

---

Updated as suggested.